htlcswitch/switch_test.go

```diff
@@ -3226,7 +3228,7 @@ func TestSwitchGetAttemptResultStress(t *testing.T) {
 		// s.Stop() happens in the middle of GetAttemptResult series.
 		// The value 10ms was found empirically - this time is needed
 		// to expose the race condition (as a crash under -race) in the
-		// unfixed version of Switch, before GoroutineManager was added.
+		// version of Switch before GoroutineManager was added.
```

Comment:
should be fixed in the previous commit that added it

Reply:
Fixed. Thanks!